tubular/mapping.py

```diff
@@ -163,13 +166,25 @@ def __init__(self, mappings, **kwargs):
         BaseMappingTransformer.__init__(self, mappings=mappings, **kwargs)
 
     def transform(self, X):
```

Comment:
One for the backlog probably but should all of these checks also be applied to the CrossColumnMapping transformer? and perhaps the others in mapping.py? (or put into BaseMappingTransformer)

Reply:
good point, will put that on the list